nimble/host/audio/services/bass/src/ble_audio_svc_bass.c

```diff
@@ -338,6 +372,19 @@ ble_svc_audio_bass_add_source(uint8_t *data, uint16_t data_len, uint16_t conn_ha
         operation.add_source.subgroups[i].metadata = &data[offset];
         offset += operation.add_source.subgroups[i].metadata_length;
         data_len -= operation.add_source.subgroups[i].metadata_length;
+
+        if (check_bis_sync(operation.add_source.num_subgroups,
+                           operation.add_source.bis_sync)) {
+            rc = BLE_HS_EINVAL;
+            ev.bass_operation_status.status = BLE_HS_EREJECT;
+            goto done;
+        }
+
+        if (data_len != 0) {
```

Comment:
does it pass a test where broadcaster has more than 1 subgroup ? I think that it will not work due to this check here

Reply:
Removed